libdrgn/dwarf_info.c

```diff
@@ -6588,15 +6604,8 @@ drgn_function_type_from_dwarf(struct drgn_debug_info *dbinfo,
 			is_variadic = true;
 			break;
 		case DW_TAG_template_type_parameter:
-			err = parse_template_parameter(dbinfo, module, &child,
-						       drgn_dwarf_template_type_parameter_thunk_fn,
-						       &builder.template_builder);
-			if (err)
-				goto err;
-			break;
 		case DW_TAG_template_value_parameter:
```

Comment:
It just occurred to me, can we have variadic template functions, too? If so, we need to handle that in this function, too.